lib/_tls_wrap.js

```diff
@@ -340,10 +340,10 @@ TLSSocket.prototype._wrapHandle = function(wrap) {
   res._secureContext = context;
   res.reading = handle.reading;
   Object.defineProperty(handle, 'reading', {
-    get: function readingGetter() {
+    get: function get() {
```

Comment:
I find that confusing to read. I will probably change my opinion as it becomes more common and I get used to seeing it. If there's a strong preference for it among others, I'll use it. But if left to my own devices, I prefer the explicit `key: value` format instead.